notebooks/MAST/TESS/beginner_tess_tap_search/beginner_tess_tap_search.ipynb

```diff
@@ -0,0 +1,375 @@
+   "metadata": {},
+   "source": [
+    "### Step 2: Getting an inventory of TESS lightcurves within the footprint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we take the footprint from the above query and find all lightcurves intersecting with this footprint, in ALL sectors. Depending on where this is in the sky, there could be responses only in the original sector, or there could be overlaps with other sectors. There would be more sector overlap near the poles, for instance. By filtering on obs_collection = TESS, we filter based on the TESS mission and exclude High Level Science Products even based on TESS.\n",
+    "\n",
+    "The format must be reformatted for our next query. We separate the shape from the list of vertices, which themselves must be comma-separated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "footprint = footprint_results['s_region'][0].decode('UTF-8')\n",
+    "footprintShape = footprint[0:footprint.find(' ')]\n",
+    "footprintVertices = footprint[footprint.find(' '):].strip().replace(' ', ', ')\n",
+    "print(footprintShape)\n",
+    "print(footprintVertices)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the footprint has been isolated and reformatted, we perform another query listing all lightcurves (minus the data validation timeseries, whose files end in _dvt) by their target name, sector, RA and Dec, as well as returning the access url for each FITS file and its estimated file size. We are doing this as an asynchronous query, which handles longer response times, just in case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "job = TAP_service.launch_job_async(\"\"\"\n",
+    "            SELECT target_name, sequenceNumber as sector, s_ra, s_dec, access_url, access_estsize, obs_id \n",
+    "            FROM dbo.caomobservation JOIN ivoa.obscore on dbo.caomobservation.observationID = ivoa.obscore.obs_id \n",
+    "            WHERE \n",
+    "            obs_collection = 'TESS' and dataproduct_type = 'timeseries' \n",
+    "            and access_url like '%lc.fits' and \n",
+    "            CONTAINS(POINT('ICRS',s_ra,s_dec),{}('ICRS', {}))=1\n",
+    "            ORDER BY obs_id\n",
+    "            \"\"\".format(footprintShape, footprintVertices))\n",
+    "\n",
+    "#uncomment the line below to see the whole query with wildcards replaced\n",
+    "#print(job.parameters)\n",
+    "TAP_results = job.get_results()\n",
+    "TAP_results          "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The returned data is in an astropy table; you can manipulate it to do more ordering or filtering. To download individual files or the whole set, you can use the access_url column, as below.\n",
```

Comment:
Suggest setting off "acess_url" in backtics or italics or something for readability.